projects/js-packages/publicize-components/src/hooks/use-anaytics-tracks-for-connected-user/README.md

```diff
@@ -0,0 +1,24 @@
+# useAnalyticsTracksForConnectedUser hook
```

Comment:
I just happened to create `useAnalytics()` that was merged yesterday. :) Maybe we could use that / expand on it in this PR instead of a new hook?

I reached out to the Reach team about this here, and they seemed interested:
p1671107976320099-slack-C02JJ910CNL

Maybe that'd be the opportunity to start using in Jetpack Social?

Reply:
Ah, nice. Yeah seems like your hook is in the same spirit!
Sounds like a good plan to update the one you added instead of adding a new one.

Do you think the check for a Jetpack connected user in the hook I added is too opinionated for the `useAnalytics` hook?

Reply:
It may be fine in `useAnalytics` as well. I must say I haven't tested with disconnected users until now, since I had assumed the Tracks library wouldn't be enqueued anyway, so nothing would happen.